pkg/webui/components/safe-inspector/index.js

```diff
@@ -232,8 +244,12 @@ export class SafeInspector extends Component {
     if (isBytes) {
       const chunks = chunkArray(data.toUpperCase().split(''), 2)
       if (!byteStyle) {
-        const orderedChunks = msb ? chunks : chunks.reverse()
-        formattedData = display = orderedChunks.map(chunk => `0x${chunk.join('')}`).join(', ')
+        if (representation === 'uint32_t') {
```

Comment:
Please introduce constants for representation types
```suggestion
        if (representation === UINT32_T) {
```